studio/schemas/documents/resource.js

```diff
@@ -125,6 +125,27 @@ export default {
         ],
       },
     },
+    {
+      name: 'displayComponent',
+      type: 'string',
+      title: 'Display Component',
+      description:
+        'What component should be used in egghead-next to display this feature?',
+      // hidden: ({document}) => document.type !== 'feature',
+      options: {
+        list: [
+          {
+            title: 'Dynamic Grid',
+            value: 'DynamicGrid',
+            default: true,
```

Comment:
`default: true` also didn't do anything though on the front end this is already a default so we could probably just remove this option? If we don't set any custom component the dynamic grid shows